vllm/model_executor/guided_decoding/outlines_logits_processors.py

```diff
@@ -18,12 +18,13 @@
 import math
 from collections import defaultdict
 from functools import lru_cache
-from typing import Callable, DefaultDict, Dict, List, Union
+from typing import Callable, DefaultDict, Dict, List, Optional, Union
 
-import torch
+import numpy as np
 from outlines.fsm.fsm import CFGFSM, FSM, RegexFSM
 from outlines.fsm.json_schema import build_regex_from_schema
 from pydantic import BaseModel
+from torch import Tensor, from_numpy, full_like, int64
```

Comment:
stylistically we would like to keep the `torch` namespace so we are using `torch.Tensor` etc...

Reply:
Sweet on it